TypeScript.py

```diff
@@ -455,21 +458,31 @@ def sendReplaceChangesForRegions(view, regions, insertString):
         endLocation = getLocationFromPosition(view, region.end())
         cli.service.change(view.file_name(), location, endLocation, insertString)
 
+def recvReloadAck(reloadResp):
+    if reloadResp.request_seq in cli.tempFileMap:
+        tmpfile = cli.tempFileMap.pop(reloadResp.request_seq)
+        if tmpfile:
+            cli.tempFileList.append(tmpfile)
+
 def getTempFileName():
-   return os.path.join(pluginDir, ".tmpbuf")
+    seq = cli.service.seq
+    if len(cli.tempFileList) > 0:
+        tmpfile = cli.tempFileList.pop()
+    else:
+        tmpfile = os.path.join(pluginDir, ".tmpbuf"+str(cli.tmpseq))
+        cli.tmpseq += 1
+    cli.tempFileMap[seq] = tmpfile
```

Comment:
Do we delete the temp files after exiting sublime? If not, because the ``tmpseq`` starts from 1 every time sublime launches, is it possible that old temp files are reused? Will that pollute anything?